src/Selenium2Driver.php

```diff
@@ -1029,40 +1029,24 @@ public function dragTo(string $sourceXpath, string $destinationXpath)
         $target = $this->findElement($destinationXpath);
 
         $this->getWebDriverSession()->moveto(['element' => $source->getID()]);
+        $this->getWebDriverSession()->buttondown();
```

Comment:
1. There seems to be some kind of mixed indentation problem: the indentation is different, at least on GitHub.Com, between `$source = $this->findElement($sourceXpath);` and `$target = $this->findElement($destinationXpath);` lines above this review comment position.
2. What about seven uncovered lines? Were they missing coverage before this PR as well?

P.S.
I'm running tests locally with coverage enabled + using https://docs.codecov.com/docs/the-codecov-browser-extension in the Web Browser.

Reply:
1. checked in the IDE and in GitHub PR and both looked fine - it only showed up in a specific commit... so I guess it's just some GH defect.
2. I don't know what is happening there - the method is now complication if-statement-free, all lines should be executed. My only guess is that it could relate to NOWDOC/HEREDOC usage. I also see the warnings/hints as github annotations (I don't have the codecov plugin though).
![{0381133C-2FD4-425D-9132-92E648DCD520}](https://github.com/user-attachments/assets/e7f992c1-279f-467b-9873-a75652f1d31b)
On the other hand, check the result of the last actions run - the codecov check passed fine:
![image](https://github.com/user-attachments/assets/85a83e41-d8a1-44c6-af2a-ab7c36f01e08)

Reply:
Ha-ha. The CodeCov website is showing PR as 100% covered: https://app.codecov.io/gh/minkphp/MinkSelenium2Driver/pull/408?dropdown=coverage&src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=minkphp .

I guess it either incorrectly reports coverage to GitHub or GitHub is showing it incorrectly.

Anyway, both of the original issues were in fact not issues.